test_coverage/test_gcovr.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+# source code: https://github.com/gcovr/gcovr/
+# docs: https://gcovr.com/en/stable/guide.html
+# install GCOVR
+# $ pip install gcovr
+
+PREFIX_ERRN="ERRN:"
+DIR_BUILD="build_gcovr"
+TIMESTAMP=$(date +"%y-%m-%d-%H:%M:%S")
+DIR_REPORT="test_coverage/report_gcovr_${TIMESTAMP}"
+# Jump to build folder
+if [ ! -d ../$DIR_BUILD ]
+  then
+    mkdir ../$DIR_BUILD
+    if [ $? -ne 0 ]
+    then
+      echo $PREFIX_ERRN "cannot make dir '"$DIR_BUILD"'"
+      exit 1
+    fi  
+fi
+cd ../$DIR_BUILD
+if [ $? -ne 0 ]
+  then
+  echo $PREFIX_ERRN "cannot change dir '"$DIR_BUILD"'"
+  exit 1
+fi
+rm -rf ./*
+# Build project with flags
+cmake -DFACELIFT_ENABLE_GCOV=ON .. && make
+# Run facelift test-cases
+examples/launch-tuner.sh
```

Comment:
I think you want to run ctest (all tests) to measure the coverage, not just a single example

Reply:
Sure